x-pack/plugins/observability_solution/apm/public/components/app/onboarding/instructions/otel_agent.tsx

```diff
@@ -244,7 +244,10 @@ export function OpenTelemetryInstructions({
                 target="_blank"
                 href="https://github.com/open-telemetry/opentelemetry-specification/blob/v1.10.0/specification/protocol/exporter.md"
               >
-                OTEL_EXPORTER_OTLP_ENDPOINT
+                {i18n.translate(
+                  'xpack.apm.openTelemetryInstructions.otelexporterotlpendpointLinkLabel',
+                  { defaultMessage: 'OTEL_EXPORTER_OTLP_ENDPOINT' }
+                )}
```

Comment:
I think this doesn't make sense to translate too. None of the options below.

@achyutjhunjhunwala what do you think?